### ISSUE TITLE
Compatibility with expo 33

### ISSUE THREAD

**Issue by leanderlee** [GetStream/stream-chat-react-native#49]
Hi there, I'm getting this warning when using your library:

```The following APIs have moved to separate packages and importing them from the "expo" package is deprecated: ImagePicker, Permissions.

1. Add correct versions of these packages to your project using:

   expo install expo-image-picker expo-permissions

   If "install" is not recognized as an expo command, update your expo-cli installation.

2. Change your imports so they use specific packages instead of the "expo" package:

 - import { ImagePicker } from 'expo' -> import * as ImagePicker from 'expo-image-picker'
 - import { Permissions } from 'expo' -> import * as Permissions from 'expo-permissions'
```

And
```
The following APIs have moved to separate packages and importing them from the "expo" package is deprecated: DocumentPicker.

1. Add correct versions of these packages to your project using:

   expo install expo-document-picker

   If "install" is not recognized as an expo command, update your expo-cli installation.

2. Change your imports so they use specific packages instead of the "expo" package:

 - import { DocumentPicker } from 'expo' -> import * as DocumentPicker from 'expo-document-picker'```

**Comment by vishalnarkhede** [comment]
Hey @leanderlee 

Can you please specify version of `expo-cli` and version of our SDK that you are using?

**Comment by leanderlee** [comment]
expo-cli: v3.0.9
```
├─ stream-chat-expo@0.1.18
│  └─ stream-chat-react-native-core@v0.1.18
```
For more info about the expo 33 migration, I believe it's explained at https://blog.expo.io/expo-sdk-v33-0-0-is-now-available-52d1c99dfe4c#16e1

**Comment by vishalnarkhede** [comment]
@leanderlee 

Yeah I saw the documentation. But weird thing is that I am not able to reproduce those warnings somehow. I tried our library in app with Expo SDK 33 as well as 34.

**Comment by leanderlee** [comment]
Not sure if this helps, but the warnings don’t appear until you try to use the module, like when you try to pick an image or file.

**Comment by agustif** [comment]
I hadn't this the issue and haven't fully tested my solution but I think this https://github.com/GetStream/stream-chat-react-native/pull/54 should fix it

**Comment by vishalnarkhede** [comment]
@agustif @leanderlee I have just released v0.2.1 of our sdk, which has compatibility for expo 33 and 34 both. Please test it and let me know the feedback or any issues! :)

**Comment by agustif** [comment]
> @agustif @leanderlee I have just released v0.2.1 of our sdk, which has compatibility for expo 33 and 34 both. Please test it and let me know the feedback or any issues! :)

Image-Picker is working for me, Document Picker is not, not sure if it's related to the same issue or not though.

I can't make it work nor in the iOS Simulator or my own iPhone. on the phone when trying for a second time I get this.

[Unhandled promise rejection: Error: Different document picking in progress. Await other document picking first.]

In the Simulator I just get an empty screen with an Expo logo in the middle after clicking browse in order to select a file.

**Comment by agustif** [comment]
> > @agustif @leanderlee I have just released v0.2.1 of our sdk, which has compatibility for expo 33 and 34 both. Please test it and let me know the feedback or any issues! :)
> 
> Image-Picker is working for me, Document Picker is not, not sure if it's related to the same issue or not though.
> 
> I can't make it work nor in the iOS Simulator or my own iPhone. on the phone when trying for a second time I get this.
> 
> [Unhandled promise rejection: Error: Different document picking in progress. Await other document picking first.]
> 
> In the Simulator I just get an empty screen with an Expo logo in the middle after clicking browse in order to select a file.

This issue has been resolved, Thank you @vishalnarkhede for all the help!

**Comment by agustif** [comment]
@leanderlee can you confirm you don't have any warnings also? Maybe this can be closed! since #58

**Comment by vishalnarkhede** [comment]
There will be some warnings on expo 32 and 33 I think. They don't break anything, everything will work smoothly. But they are currently unavoidable (since we want to support expo 32, 33 and 34), so can't do much about it. They only exist in dev mode.

**Comment by leanderlee** [comment]
@agustif I can confirm it's fixed now in 0.2.4. Thank you!
